content/en/continuous_integration/setup_tests/swift.md

```diff
@@ -39,37 +43,121 @@ There are two ways of installing the testing framework:
+**Default**: `(empty)`<br/>
+**Examples**: `pub0zxxxyyyxxxyyxxxzzxxyyxxxyyy`
+
+`DD_SERVICE`
+: Name of the service or library under test.<br/>
+**Default**: `(Repository name)`<br/>
+**Example**: `my-marvelous-app`
+
+`DD_ENV`
+: Name of the environment where tests are being run. You should set this value to $(DD_ENV) so you can use an environment variable for setting it<br/>
+**Default**: `(empty)`<br/>
+**Recommended**: `$(DD_ENV)`<br/>
+**Examples**: `ci`, `local`
+
+`SRCROOT`
+: This is the path to the project SRCROOT environment variable. Best option is using `$(SRCROOT)` for the value, so it uses Xcode internal variable<br/>
+**Default**: `(empty)`<br/>
+**Recommended**: `$(SRCROOT)`<br/>
+**Example**: `/Users/ci/source/MyApp`
+
+`DD_ENDPOINT`
+: This is not needed if your company is using US endpoint<br/>
+**Default**: `us`<br/>
+**Examples**: `eu1`
+
+### Configuring your CI service
 
+You must also set environment variables for your CI service. Note that Git information is automatically collected if you use a simulator to perform your MacOS, iOS, or tvOS tests. However, If you do testing on physical devices, you must also set the additional Git-related environment variables. You can find a list of the CI services and the environment variables at the end of this document, in the section [CI Services](#CI-Services)
 
-Optionally, set these environment variables also:
+### Running tests
 
-| Environment variable | Value                                      |
-| -------------------- | ------------------------------------------ |
-| `DD_ENV `            | The environment you want to report         |
-| `DD_SERVICE`         | The name of the service you want to report |
+After installation, you can run your tests as you normally do, for example using the `xcodebuild test` command. Tests, network requests, and application logs will be instrumented automatically. You can pass your environment variables when running your tests in the CI, e.g:
 
+{{< code-block lang="bash" >}}
+DD_TEST_RUNNER=1 DD_ENV=ci xcodebuild -project "MyProject.xcodeproj" -scheme "MyScheme" -destination "platform=macOS,arch=x86_64" test
+{{< /code-block >}}
 
-### Configuring your CI service
+### UI tests
+
+For UITests, both the test target and the application running from the UITests must link with the framework. Environment variables need to be set only in the test target, because the framework automatically injects these values to the application.
+
+## Additional optional configuration
+
+For the following configuration settings:
+ - `Boolean` variables can use any of: `1`, `0`, `true`, `false`, `YES`, or `NO`
+ - `String` list variables accept a list of elements separated by `,` or `;`
+
+### Disabling auto-instrumentation
+
+The framework automatically captures the most information possible, but for some situations or tests this can be counter-productive. You can disable some of the auto-instrumentation for all the tests by setting environment variables:
```

Comment:
```suggestion
The framework enables auto-instrumentation of all supported libraries, but in some cases this might not be desired. You can disable auto-instrumentation of certain libraries by setting the following environment variables:
```